artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
```

Comment:
`"music.csv"` feels like a constant/module attribute to me: http://elixir-lang.org/getting-started/module-attributes.html#as-constants